onnx/test/shape_inference_test.py

```diff
@@ -262,10 +288,12 @@ def test_transpose_preexisting(self, _, version) -> None:
 
     @parameterized.expand(all_versions_for("Transpose"))
     def test_transpose_scalar(self, _, version) -> None:
+        self.skipIf(version == 1, "Transpose a scale is not supported in Version 1")
```

Comment:
nit: Why not just modify the input to `@parameterized.expand([x for x in all_versions_for("Transpose") if x > 1])` ? Better still, I think it would be simpler to extend the function `all_versions_for` to take an optional lower_bound as a parameter, so we can write `@parameterized.expand(all_versions_for("Transpose", from=2))`